_data/amit2020cs/amit2020cs.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Amit Kumar Singh 
+institution: Guru Govind Singh Indraprasth University
```

Comment:
Possible typo for the name of institution?
From: Guru Govind Singh Indraprasth University
To: Guru Gobind Singh Indraprastha University